src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -42,6 +42,7 @@ class TelemetrySession(object):  # pylint: disable=too-many-instance-attributes
     extension_version = None
     feedback = None
     extension_management_detail = None
+    extension_events = []
```

Comment:
Do we need to have a list cache the extension events? Can the events directly added to the `self.events` in the `add_extension_events`?

Reply:
Fixed! Now the extension events are directly appended to `self.events`